python/py-ukpostcodeparser/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim: fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-ukpostcodeparser
+set real_name       UkPostcodeParser
+version             1.1.2
+revision            0
+
+platforms           darwin
+supported_archs     noarch
+license             MIT
+maintainers         {outlook.de:judaew @judaew} openmaintainers
+
+description         UK Postcode parser
+long_description    {*}${description}
+homepage            https://github.com/hamstah/ukpostcodeparser
+
+master_sites        pypi:U/${real_name}/
+distname            ${real_name}-${version}
```

Comment:
when you set `python.rootname` both the `master_sites` and `distname` will be set correctly by the PG so please remove these lines.